### DISCUSSION TITLE
Batch segment splitting

### DISCUSSION THREAD

**Discussion by Bearmancer** [mifi/lossless-cut#2268] [Q&A]
I have a folder with several files that are each more than 20m. I want to run a single command that automatically splits them at the 20m mark. At the moment I have to do this for every file individually. Is this possible to automate?

**Comment by mifi** [comment]
See [FAQ](https://github.com/mifi/lossless-cut/blob/master/issues.md#faq):

> - Can LosslessCut do a batch conversion of multiple files?
>   - Most likely not, but if you download [FFmpeg](https://ffmpeg.org/) on your computer and then ask an AI exactly what you want, it might be able to create a bash script for you. Example ChatGPT prompt: `Can you help me write a bash script that for each .mp4 file in a specified folder, losslessly cuts off the first 10 seconds from each file?` - You can also feed the AI commands from "Last FFmpeg commands" page in LosslessCut as an example.

**Reply by Bearmancer** [reply]
Okay. I did it but it was slow as molasses the first time. It was quite fast the second time so issue solved. Thanks.
